### PR TITLE
Add StratisTest network to NStratis

### PR DESCRIPTION
**Pull request by zeptin**
Currently this network is defined in the StratisBitcoinFullNode repository. It should be in NStratis so that it can be used in all dependent projects.

### CODE REVIEW

File: NBitcoin/Networks.cs [hunk 1]
```diff
@@ -199,125 +210,111 @@ private static Network InitStratisMain()
...
+				.SetRPCPort(26174)
+				.SetBase58Bytes(Base58Type.PUBKEY_ADDRESS, new byte[] { (65) })
+				.SetBase58Bytes(Base58Type.SCRIPT_ADDRESS, new byte[] { (196) })
+				.SetBase58Bytes(Base58Type.SECRET_KEY, new byte[] { (65 + 128) })
+				.SetBase58Bytes(Base58Type.ENCRYPTED_SECRET_KEY_NO_EC, new byte[] { 0x01, 0x42 })
+				.SetBase58Bytes(Base58Type.ENCRYPTED_SECRET_KEY_EC, new byte[] { 0x01, 0x43 })
+				.SetBase58Bytes(Base58Type.EXT_PUBLIC_KEY, new byte[] { (0x04), (0x88), (0xB2), (0x1E) })
+				.SetBase58Bytes(Base58Type.EXT_SECRET_KEY, new byte[] { (0x04), (0x88), (0xAD), (0xE4) })
+
+#if !NOSOCKET
```
Comment (bokobza) [inline NBitcoin/Networks.cs:254]: why is this needed here?

**Review by bokobza — COMMENTED** [review]
Makes sense to add this method here.
We also need to remove it from the Breeze.Daemon.Program class

File: NBitcoin/Networks.cs (hunk 1 above)
Comment (bokobza) [reply NBitcoin/Networks.cs:254]: ok, the other netwroks are doing this as well.

**Comment by bokobza** [comment]
As discussed, this needs to be followed up by removing the use of the duplicated method in the StratisD and the Breeze.Daemon projects.
@dangershony can you please push this in, I don't have rights to merge it? Thanks

**Review by dangershony — APPROVED** [review]

**Comment by dangershony** [comment]
Awesome yeah remember to delete it from the fullnode (and Breeze)
